static/context-debug.js

```diff
@@ -0,0 +1,29 @@
+// Override the Karma setup for local debugging
+window.__karma__.info = function(info) {
```

Comment:
This file doesn't seem to be linted as  `function()` should be `function ()` according to our linting rules.